src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2689,6 +2691,9 @@ public ConstructorEdit(INamedTypeSymbol oldType)
                                 newSymbol = GetSymbolForEdit(newModel, edit.NewNode, edit.Kind, editMap, out var newIsAmbiguous, cancellationToken);
                                 if (newSymbol == null || !processedSymbols.Add(newSymbol))
                                 {
+                                    // Field declarations have attributes applied to them, but wont return symbols from the above, so need special handling.
```

Comment:
>declarations [](http://example.com/codeflow?start=45&length=12)

This might need more explanation - GetSymbolEdit returns field symbol if the initializer is updated, right? Did you mean when only attributes are updated on a field?

Reply:
Also, there are two conditions here: `newSymbol == null` and `!processedSymbols.Add(newSymbol)`. Seems like this only applies to the first condition?